Tests/App/security.php

```diff
@@ -9,10 +9,12 @@
  * file that was distributed with this source code.
  */
 
+use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\Security\Core\Security;
 use Symfony\Component\Security\Http\Controller\UserValueResolver;
 
-if (method_exists(Security::class, 'getUser') && !class_exists(UserValueResolver::class)) {
+if (isset($container) && method_exists(Security::class, 'getUser') && !class_exists(UserValueResolver::class)) {
```

Comment:
It's config file, maybe we need: https://packagist.org/packages/phpstan/phpstan-symfony ?

Reply:
I thought about it. But is it worth it for this one place? If we add more php config files for Symfony then maybe